modules/xfem/src/base/XFEMGeometricCut2D.C

```diff
@@ -24,65 +24,74 @@ XFEMGeometricCut2D::~XFEMGeometricCut2D()
 }
 
 bool
+XFEMGeometricCut2D::active(Real time)
+{
+  return cutFraction(time) > 0;
+}
+
+bool
 XFEMGeometricCut2D::cutElementByGeometry(const Elem* elem, std::vector<CutEdge> & cut_edges, Real time)
 {
-  //Use the algorithm described here to determine whether edges are cut by the cut line:
-  //http://stackoverflow.com/questions/563198/how-do-you-detect-where-two-line-segments-intersect
+  // Use the algorithm described here to determine whether edges are cut by the cut line:
+  // http://stackoverflow.com/questions/563198/how-do-you-detect-where-two-line-segments-intersect
```

Comment:
Seem like we have some potential code duplication here:
https://github.com/idaholab/moose/blob/devel/framework/src/utils/LineSegment.C